### ISSUE TITLE
Go get - too many errors

### ISSUE THREAD

**Issue by lenisko** [gogits/gogs#3683]
- Gogs version (or commit ref): https://github.com/gogits/gogs/commit/7e15ff9486cde36b76f3d39b7db23f89278acb7d
- Git version: 2.7.4
- Go version: go1.7.1 linux/amd64
- Operating system: Ubuntu 16.04.1

## Description
There's a problem when installing using sources

```
$ go get -u github.com/gogits/gogs
# github.com/gogits/gogs/models
go/src/github.com/gogits/gogs/models/access.go:84: cannot use x (type *xorm.Engine) as type Engine in argument to accessLevel:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/access.go:94: cannot use x (type *xorm.Engine) as type Engine in argument to hasAccess:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/access.go:245: cannot use x (type *xorm.Engine) as type Engine in argument to r.recalculateAccesses:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/action.go:194: cannot use x (type *xorm.Engine) as type Engine in argument to newRepoAction:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/action.go:217: cannot use x (type *xorm.Engine) as type Engine in argument to renameRepoAction:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/action.go:568: cannot use x (type *xorm.Engine) as type Engine in argument to transferRepoAction:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/action.go:586: cannot use x (type *xorm.Engine) as type Engine in argument to mergePullRequestAction:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/issue.go:154: cannot use x (type *xorm.Engine) as type Engine in argument to issue.loadAttributes:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/issue.go:231: cannot use x (type *xorm.Engine) as type Engine in argument to i.hasLabel:
        *xorm.Engine does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/issue.go:312: cannot use e (type *xorm.Session) as type Engine in argument to issue.getLabels:
        *xorm.Session does not implement Engine (wrong type for Where method)
                have Where(interface {}, ...interface {}) *xorm.Session
                want Where(string, ...interface {}) *xorm.Session
go/src/github.com/gogits/gogs/models/issue.go:312: too many errors
```

**Comment by wxd237** [comment]
I have the same question!

**Comment by chaosbunker** [comment]
I am having the same issue upgrading from source

**Comment by kikimor** [comment]
:+1:

**Comment by 0xbaadf00d** [comment]
@lenisko you dont have filled all fields on the issue reporting template. What DBMS are you using ? Have you try with a different DBMS ?

**Comment by hiliev** [comment]
Same problem here. Go version is go1.7.1 freebsd/amd64.

**Comment by jeppech** [comment]
Is this due to, xorm making some breaking changes to its API?

**Comment by andreynering** [comment]
> Is this due to, xorm making some breaking changes to its API?

Yes, see discussion on [Gitter](https://gitter.im/gogits/gogs) and fix here: https://github.com/gogits/gogs/pull/3685

**Comment by jeppech** [comment]
Cool 👍 !

**Comment by Unknwon** [comment]
Fixed by #3685
